public/Player.js

```diff
@@ -36,11 +40,34 @@ export default class Player {
     this.domElement = htmlPlayersContainer.querySelector(`#${this.name}`);
   }
 
+  playerBetValurHtml() {
```

Comment:
This method is not being used, does it need to be deleted? Also there is a typo:
```suggestion
  playerBetValueHtml() {
```